doc/man3/EVP_EncryptInit.pod

```diff
@@ -409,8 +409,8 @@ The ChaCha20 stream cipher. The key length is 256 bits, the IV is 96 bits long.
 
 Authenticated encryption with ChaCha20-Poly1305. Like EVP_chacha20() the key is
 256 bits and the IV is 96 bits. This supports additional authenticated
-data (AAD) and produces a 128 bit authentication tag. The L</GCM and OCB modes>
-section below applies.
+data (AAD) and produces a 128 bit authentication tag. See the
+"GCM and OCB modes" section, below.
```

Comment:
Because perldoc complained it couldn't find it.  And since it appears on the very next line, it was easiest to just remove the section.